src/Symfony/Component/DependencyInjection/ContainerInterface.php

```diff
@@ -51,7 +48,7 @@ public function set($id, $service);
      *
      * @see Reference
      */
-    public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE);
+    public function get($id, int $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE);
```

Comment:
Note: We cannot add `string $id` because we would violate the PSR interface otherwise.